ocaml/xapi-idl/network/network_interface.ml

```diff
@@ -34,19 +34,26 @@ let comp f g x = f (g x)
 let ( ++ ) f g x = comp f g x
 
 let netmask_to_prefixlen netmask =
```

Comment:
Is this difficult to convert to accept ipv6 addresses as well?

In general, I would prefer it instead of strings, networkmanager handled Ipaddr.t, that's a bigger change, however

Reply:
I was under the impression IPv6 doesn't have netmasks, only CIDR notation prefixes

Reply:
I'm not sure I understand - I kept the interface of both functions the same, so they are `string -> int` and `int -> string`. `Ipaddr.t` is a detail of internal implementation I keep hidden and transform the input to myself.